apiconcepts/integration/what_you_can_do_with_the_integration_API.md

```diff
@@ -3,10 +3,11 @@ What you can do with the Integration API
 
 This page provides a quick overview of what you can do with the <Var:ProductName> Integration API.
 
-The <Var:ProductName> Integration API enables third-party developers to:
+As you can see in the diagram below, the <Var:ProductName> Integration API enables third-party developers to provide custom plug-ins in order to:
```

Comment:
Please change this to:
The following diagram exemplifies how ... API  enables you to create custom plug-ins in order to: